RequestService/RequestService.Core/Services/ManageRequestsService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using RequestService.Core.Interfaces.Repositories;
+using System;
+
+namespace RequestService.Core.Services
+{
+    public class ManageRequestsService : IManageRequestsService
+    {
+        private readonly IRepository _repository;
+
+        public ManageRequestsService(IRepository repository)
+        {
+            _repository = repository;
+        }
+        
+        public void ManageRequests()
+        {
+            _repository.UpdateInProgressFromAccepted();
+            _repository.UpdateJobsToDoneFromInProgressOrAccepted();
```

Comment:
On reflection, let's make the second method just **UpdateJobsToDoneFromInProgress**.

Any Accepted jobs will be advanced to InProgress by UpdateInProgressFromAccepted, so there shouldn't be any left.